### DISCUSSION TITLE
Server Error while trying to import RACKs

### DISCUSSION THREAD

**Discussion by benek984** [netbox-community/netbox#12297] [Help Wanted!]
Getting below error while trying to import few RACKs

Example of my file 
```
Name,Site,Status,Role,Height,Width
UK-CH-RACK01,Chesterford Office,Active,,49U,19 inches
UAE-DU-RACK01,Dubai ICS,Active,,49U,19 inches
```

![image](https://user-images.githubusercontent.com/62546477/233092207-3076abc5-1e9e-49a8-9150-d6551316eb4f.png)

**Comment by candlerb** [comment]
I can reproduce the exception with Netbox v3.4.8, so that ought to be fixed to give a proper data validation error.

Firstly: change the header row to

```
name,site,status,role,u_height,width
```

and try again. Now, assuming that the sites you have named do actually exist, then you should get other errors, properly shown this time (if you still get an excecption, hit the Back button on your browser to see the flash messages).

Various things you'll need to change:
* status "Active" to "active"
* u_height "49U" to "49"
* width "19 inches" to "19"

This is certainly a rather terrible user experience.

**Reply by benek984** [reply]
Thank you for you suggestion I will give it a go. However, capital first letters are done by Netbox when exporting file. I added few Racks manually then exported them so I can have template file. In this case I think Netbox should accept something that has produce themself.

**Reply by candlerb** [reply]
> However, capital first letters are done by Netbox when exporting file

Sorry, but export and import CSV formats in Netbox are completely different and incompatible, so you cannot use an export as a template for an import. (It has been discussed many times before)

**Comment by jeremystretch** [comment]
There's a table directly below the import form which lists all of the supported attributes (many of which are _not_ included in the default export template) and their formats. Please refer to this instead of trying to guess from exported data, which is intended to be used in systems outside of NetBox.
